windows/vbs_enable_disable/vbs_enable_test.yml

```diff
@@ -6,60 +6,95 @@
+      include_tasks: ../utils/win_enable_vbs_vm.yml
       vars:
         win_enable_vbs: true
-    - include_tasks: ../../common/vm_set_power_state.yml
+    - name: "Power on VM"
+      include_tasks: ../../common/vm_set_power_state.yml
       vars:
         vm_power_state_set: "powered-on"
-    - include_tasks: ../utils/win_update_inventory.yml
-    # Check VM VBS status
-    - include_tasks: ../../common/vm_get_vbs_status.yml
+    - name: "Update in-memory inventory after VM power on"
+      include_tasks: ../utils/win_update_inventory.yml
+    - name: "Get VM VBS status"
+      include_tasks: ../../common/vm_get_vbs_status.yml
     - name: "Check VM VBS status after enable"
       ansible.builtin.assert:
         that:
           - vm_vbs_enabled is defined
           - vm_vbs_enabled | bool
-        fail_msg: "VM VBS status is not enabled after enabling it."
+        fail_msg: "VM VBS status is '{{ vm_vbs_enabled | default('') }}', not enabled after enabling it."
   when: not vm_vbs_enabled_before
 
-- name: "VM VBS is enabled"
-  block:
-    # Get VBS status in guest OS
-    - include_tasks: ../utils/win_get_vbs_guest.yml
-    - name: "Set fact of HVCI and VBS running status in guest before enable"
-      ansible.builtin.set_fact:
-        guest_vbs_enabled_before: true
-      when:
-        - win_vbs_status_guest | int == 2
-        - "'2' in win_vbs_running_service"
-  when: vm_vbs_enabled_before
+- name: "Get Device Guard available security properties in guest OS"
+  include_tasks: ../utils/win_get_dg_security_properties.yml
+
+- name: "Enable VBS and security services in guest OS"
+  include_tasks: ../utils/win_enable_vbs_guest.yml
+
+- name: "Get VBS status and running security services"
+  include_tasks: ../utils/win_get_vbs_guest.yml
 
-# Enable VBS in guest OS if HVCI is not running or VBS is not running
-- name: "Enable VBS in guest OS"
+# AvailableSecurityProperties:
+# 0. 	If present, no relevant properties exist on the device.
+# 1. 	If present, hypervisor support is available.
+# 2. 	If present, Secure Boot is available.
+# 3. 	If present, DMA protection is available.
+# 4. 	If present, Secure Memory Overwrite is available.
+# 5. 	If present, NX protections are available.
+# 6. 	If present, SMM mitigations are available.
+# 7. 	If present, MBEC/GMET is available.
+# 8. 	If present, APIC virtualization is available.
+#
+- name: "Handle known issue"
   block:
-    - include_tasks: ../utils/win_enable_vbs_guest.yml
-    - include_tasks: ../utils/win_get_vbs_guest.yml
-  when: not guest_vbs_enabled_before
+    - name: "Known issue - NX protections are not present in AvailableSecurityProperties on ESXi 7.0.3"
+      ansible.builtin.debug:
+        msg:
+          - "NX protections are not present in guest OS 'AvailableSecurityProperties' issue exists on ESXi 7.0.3 build {{ esxi_build }}, which is fixed in ESXi 7.0U3L patch build 21424296. Please refer to KB article: https://kb.vmware.com/s/article/91199."
```

Comment:
The first sentence has two verbs 'are' and 'exists'. I think here can be changed as:
```
The issue of NX protections are not present in guest OS 'AvailableSecurityProperties' exists from ESXi 7.0.3 build ...
```